### ISSUE TITLE
[IDEA] Implement a power system

### ISSUE THREAD

**Issue by coderCleric** [coderCleric/Poltergeist#8]
Thinking of implementing a power system to help control spamming of items, and wanted to get a vibe on it here. The idea is to do it kinda like this:
- Ghosts have a certain amount of maximum power, and recharge a certain amount of power per second.
- Interacting with objects requires the consumption of a certain amount of power, depending on the action (honking a horn might be 5, messing with a door might be 10, etc.)
- Maximum power is inversely proportional to the percentage of players still alive (so more players dying will give the ghosts more power to play with)

With this, I could even add certain interactions that have a high power cost, rendering them unusable until enough players are dead. Overall, this would really open the door to adding more interactions, since things wouldn't be quite so spammable.

Basically just making this issue to get feedback on the idea.

**Comment by coderCleric** [comment]
Resolved in 0.3.0.
